bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/DeviceDeserializer.java

```diff
@@ -0,0 +1,35 @@
+package com.bugsnag.android;
+
+import java.util.Map;
+
+class DeviceDeserializer implements MapDeserializer<DeviceWithState> {
+
+    @Override
+    public DeviceWithState deserialize(Map<String, Object> map) {
+        DeviceBuildInfo buildInfo = new DeviceBuildInfo(
+                MapUtils.<String>getOrThrow(map, "manufacturer"),
+                MapUtils.<String>getOrThrow(map, "model"),
+                MapUtils.<String>getOrThrow(map, "osVersion"),
+                MapUtils.<Integer>getOrThrow(map, "apiLevel"),
+                MapUtils.<String>getOrThrow(map, "osBuild"),
+                MapUtils.<String>getOrThrow(map, "fingerprint"),
+                MapUtils.<String>getOrThrow(map, "tags"),
+                MapUtils.<String>getOrThrow(map, "brand"),
+                MapUtils.<String[]>getOrThrow(map, "cpuAbis")
+        );
+
+        String time = MapUtils.getOrNull(map, "time");
```

Comment:
Can be `null` but `DateUtils.fromIso8601` is marked non-null - it should probably end up setting it to `null` rather than error

Reply:
Added a null check to avoid this throwing an exception.